airbyte-integrations/connectors/source-mssql/src/testFixtures/java/io/airbyte/integrations/source/mssql/MsSQLTestDatabase.java

```diff
@@ -28,7 +33,7 @@ public class MsSQLTestDatabase extends TestDatabase<MSSQLServerContainer<?>, MsS
 
   static private final Logger LOGGER = LoggerFactory.getLogger(MsSQLTestDatabase.class);
 
-  static public final int MAX_RETRIES = 60;
+  static public final int MAX_RETRIES = 240;
```

Comment:
This deserves commentary. As in, "we tried having this at 60, so 1 minute, and found this to be unwise, blah blah blah..."